assets/chat/css/style.scss

```
@@ -1569,7 +1569,12 @@ body.pref-fontscale[data-fontscale='10'] {
     display: flex;
     align-items: center;
   }
+  .opt:first-child:nth-last-child(n + 10) .opt-info,
+  .opt:nth-last-child(n + 10) ~ .opt .opt-info {
```

Any particular reason you went with
```
.opt:nth-last-child(n + 10) ~ .opt .opt-info
```
rather than
```
.opt:first-child:nth-last-child(n + 10) ~ .opt .opt-info
```
for the second selector? I think the effect is the same, but that threw me off a bit.

---

Yeah, I tried doing it that way but for some reason it wasn't applying it correctly for me, I'm not sure why.

---

How wasn't it applying correctly specifically? Seemingly no issues for me in Chrome 106 on macOS 12.6.1.

---

Hmmm, if I remember correctly the css rule wasn't applying to the first element. But I could be mistaken as I tried it again just now and it worked.

---

Great! Then let's go ahead and update this selector and I'll merge. I prefer this one because it's easier to understand.
```suggestion
  .opt:first-child:nth-last-child(n + 10) ~ .opt .opt-info {
```